Dockerfile

```diff
@@ -1,17 +1,23 @@
-FROM ghcr.io/yannh/kubeconform:v0.6.3 as kubeconform
-FROM hadolint/hadolint:v2.12.0-alpine as hadolint
-FROM koalaman/shellcheck:v0.9.0 as shellcheck
-FROM rhysd/actionlint:1.6.25 as actionlint
+# syntax=docker/dockerfile:1
```

Comment:
The `platform` work here and in the `Taskfile.yml` files was necessary for my local testing/confirmation that the goat works.

Reply:
See more info in https://docs.docker.com/build/building/multi-platform/#building-multi-platform-images and https://taskfile.dev/usage/#platform-specific-tasks-and-commands